lib/kastlex/metadata_cache.ex

```diff
@@ -28,6 +28,15 @@ defmodule Kastlex.MetadataCache do
     {:ok, brokers}
   end
 
+  def existing_topic?(t, p) do
```

Comment:
no, I created it first for message_controller but then found an easier way to verify if a given topic/partition is valid (via brod_client's cache). But then decided to leave this function as is.

Yes, `existing_partition` is probably a better name.